README.md

```diff
@@ -10,81 +10,86 @@ The frontend apps can easily be deployed to any Jamstack-friendly web host such
-   - Add `<your-app-name>` to the `workspaces.packages` list
-   - Add yarn scripts for your new app
-     - ex build script: `"dev:app:<myapp>": "wait-on \"http://localhost:${PORT:-3001}/\" && cd sites/<your-app-name> && yarn dev"`
-     - ex test script: `"test:app:<myapp>": "wait-on \"http://localhost:${PORT:-3001}/\" && cd sites/<your-app-name> && yarn test"`
-1. Run `yarn install` and now you should be able to build your app with `yarn dev:app:<myapp>`
+Approximately weekly or as our roadmap requires us to, we will merge dev to master and then update our jurisdictional branches to get our changeset on a staging environment. Once that has been QA-ed we will publish to our production environment.
 
-### Versioning
-
-We are using [lerna](https://lerna.js.org/) as a package versioning tool. It helps with keeping multiple package versions in sync for the entire monorepo.
+`ui-components` is currently released on an ad-hoc basis, but we will soon be implementing a more frequent automatic release.
 
 ## Contributing
 
 Contributions to the core Bloom applications and services are welcomed. To help us meet the project's goals around quality and maintainability, we ask that all contributors read, understand, and agree to these guidelines.
+
+### Issue tracking
+
+Our development tasks are managed through GitHub issues and any development (in the vast majority of cases) should be tied to an issue. Even if you don't plan on implementing an issue yourself, please feel free to submit them if you run into issues. Before creating an issue, check first to see if one already exists. When creating an issue, give it a descriptive title and include screenshots if relevant. Please don't start work on an issue without checking in with the Bloom team first as it may already be in development! If you have questions, feel free to tag us on issues (@seanmalbert, @emilyjablonski) and note that we are also using GitHub discussions.
+
+### Pull Requests
+
+Pull requests are opened to the dev branch, not to master. When opening a pull request please fill out the entire pull request template which includes tagging the issue your PR is related to, a description of your PR, indicating the type of change, including details for the reviewer about how to test your PR, and a testing checklist. Additionally, officially link the issue in GitHub's right-hand panel.
+
+Every PR needs to manually update our changelog. Find the relevant section (General, Frontend, Backend, UI Components) and subsection (Added, Changed, Fixed) and add a short description of your change followed by a link to the PR and your name (- Description Here ([#1234](https://github.com/bloom-housing/bloom/pull/1234)) (Your Name)). If it is a breaking change, please include **Breaking Change** and some notes below it about how to migrate.
+
+It is not required, but adding the ready for review label onto your PR when it is ready will help surface it to our internal team, as will assigning specific reviewers.
```

Comment:
Should we make it required, in terms of process? And should it also say something about the WIP tag? I'd also like something for if they're the reviewer.

Something like:
If you are reviewing a PR, try not to leave only comments. If the PR requires further discussion or changes, mark it with `Request Changes`. If a PR looks good to you or there are changes request that won't require an additional review, please mark it with `Approved`. This helps other reviewers better understand the state of PRs at the list view.